README.md

```diff
@@ -1,3 +1,296 @@
+  * [Azure Kubernetes Service (AKS)](https://docs.microsoft.com/en-us/azure/aks/) on Microsoft Azure
+* Your base cluster needs at least 4 nodes with a size of 8GB for each node
+* You need a service account for the virtual machine instance of your IaaS provider where your Kubernetes version runs
+* You need to have permissions to access your base cluster's private key
+* You are connected to your Kubernetes cluster (environment variable `KUBECONFIG` is set)
+  * [Viewing kubeconfig (GKE)](https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-access-for-kubectl#viewing_kubeconfig)
+  * [Create a kubeconfig for Amazon EKS](https://docs.aws.amazon.com/eks/latest/userguide/create-kubeconfig.html)
+  * [Use Azure role-based access controls to define access to the Kubernetes configuration file in Azure Kubernetes Service (AKS)](https://docs.microsoft.com/en-us/azure/aks/control-kubeconfig-access)
+
+## Procedure
+
+To install Gardener in your base cluster, a command line tool [sow](https://github.com/gardener/sow) is used. It depends on other tools to be installed. To make it simple, we have created a Docker image that already contains `sow` and all required tools. To execute `sow` you call a [wrapper script](https://github.com/gardener/sow/tree/master/docker/bin) which starts `sow` in a Docker container (Docker will download the image from [eu.gcr.io/gardener-project/sow](http://eu.gcr.io/gardener-project/sow) if it is not available locally yet). Docker executes the sow command with the given arguments, and mounts parts of your file system into that container so that `sow` can read configuration files for the installation of Gardener components, and can persist the state of your installation. After `sow`'s execution Docker removes the container again.
+
+1. Clone the `sow` repository and add the path to our [wrapper script](https://github.com/gardener/sow/tree/master/docker/bin) to your `PATH` variable so you can call `sow` on the command line.
+
+    ```bash
+    # setup for calling sow via the wrapper
+    git clone "https://github.com/gardener/sow"
+    cd sow
+    export PATH=$PATH:$PWD/docker/bin
+    ```
+
+1. Create a directory `landscape` for your Gardener landscape and clone this repository into a subdirectory called `crop`:
+
+    ```bash
+    cd ..
+    mkdir landscape
+    cd landscape
+    git clone "https://github.com/gardener/garden-setup" crop
+    ```
+
+1. If you don't have your `kubekonfig` stored locally somewhere yet, download it. For example, for GKE you would use the following command:
+
+    ```bash
+    gcloud container clusters get-credentials <your_cluster> --zone <your_zone> --project <your_project>
+    ```
+
+1. It is recommended to save your `kubeconfig` in your `landscape` directory. For the remaining steps we will assume that you saved it using file path `landscape/kubeconfig`.
```

Comment:
I'd probably rephrase this. This way, it sounds like it would be a good idea to put the kubeconfig in the landscape directory, but it's not necessary. Due to the filesystem mounting in the docker container, the kubeconfig actually *has to be* somewhere inside the landscape folder (or within a parent directory of it). For simplicity, we should just write that the kubeconfig needs to be somewhere within the landscape directory, so no-one tries to put it elsewhere and wonders why the setup doesn't work.

Reply:
ok, it now says: "Save your `kubeconfig` somewhere in your `landscape` directory, leaving out the "It is recommended.